.github/workflows/eks-ci.yml

```diff
@@ -16,16 +16,16 @@ on:
       - "[0-9]+.[0-9]+"
 
 env:
-  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID_TEST_ACC }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY_TEST_ACC }}
   GITHUB_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   CONTAINER_SUFFIX: ${{ github.run_id }}
   TEST_TARGETS_DEFAULT: '["pre_merge"]'
   AWS_REGION: eu-west-2
   REPORTS_DIR: tests/allure/results/
   KUBE_NAMESPACE: kube-system
   CI_ELASTIC_AGENT_DOCKER_TAG: "8.7.0-SNAPSHOT"
-  CI_ELASTIC_AGENT_DOCKER_IMAGE: "704479110758.dkr.ecr.eu-west-2.amazonaws.com/elastic-agent"
+  CI_ELASTIC_AGENT_DOCKER_IMAGE: "391946104644.dkr.ecr.eu-west-2.amazonaws.com/elastic-agent"
```

Comment:
should we bump the docker tag?

Reply:
also, why snapshot?

Reply:
Yes, we can update the docker tag to the latest version. These variables (`CI_ELASTIC_AGENT_DOCKER_TAG, CI_ELASTIC_AGENT_DOCKER_IMAGE )` are used to upload the latest agent docker snapshot with the latest Cloudbeat to the ECR. Initially, we had plans to implement integration tests with a standalone agent for EKS, just like we did for K8s. However, due to prioritization, this implementation was not given enough priority, and the uploaded image is currently not being used.